support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -43,7 +43,8 @@ class CustomActionBuilders(
       ("returnUrl" -> s"$supportUrl$path") &
       idSkipConfirmation &
       idSkipValidationReturn &
-      "clientId" -> clientId).toString
+      "clientId" -> clientId &
+      createAuthenticationComponentEventTuple(SigninRedirectForSupporters)).toString
```

Comment:
probably a bit overkill having a special mechanism just to create a fixed string, would this do the same thing?
```suggestion
      "componentEventParams" -> "componentType=identityauthentication&componentId=signin_redirect_for_supporters"
    ).toString
```

Reply:
I agree 😄 
I reused code from projects where we have multiple event IDs that need to be encoded in different places. But you're right, it's not necessary here.